schema.go

```diff
@@ -320,6 +320,16 @@ var Schema = schema.NewTypedScopeSchema[*Config](
 				nil,
 				nil,
 			).Disable(notImplemented),
+			"Privileged": schema.NewPropertySchema(
+				schema.NewBoolSchema(),
+				schema.NewDisplayValue(schema.PointerTo("Privileged"), schema.PointerTo("Execute container process without any further \"security\" lockdown"), nil),
```

Comment:
I'd recommend borrowing [Podman's text:](https://docs.podman.io/en/latest/markdown/podman-run.1.html#privileged)
> "Execute container process without security features that isolate the container from the host"

Reply:
did it

Reply:
I eventually found that you'd lifted your original text from their blog post, so...yeah.  (But, I didn't feel like "lockdown" and the scare-quotes around "security" were the best for our documentation.)